alerts/openshift-virtualization-operator/PersistentVolumeFillingUp.md

```diff
@@ -0,0 +1,154 @@
+<!--
+Original source from KubePersistentVolumeFillingUp
+
+https://github.com/prometheus-operator/runbooks/blob/main/content/runbooks/kubernetes/KubePersistentVolumeFillingUp.md
```

Comment:
you might not know but https://github.com/prometheus-operator/runbooks is barely maintained.

I wonder why we'd need another OCP runbook while https://github.com/openshift/runbooks/blob/master/alerts/cluster-monitoring-operator/KubePersistentVolumeFillingUp.md already exists?
Does kubevirt/CNV deploy a dedicated `PersistentVolumeFillingUp` alerting rule?

cc @danielmellado @jan--f

Reply:
OCP is limited for openshift- and kube- namespaces
we have requests for a more generic alert for volume running out of space in CNV

Reply:
Except for the namespace matcher, what are the other differences between the 2 rules?

Both alert names being very similar may create confusion IMHO (I was confused at first). At the very least, I'd expect the CNV alert name to have a prefix (https://github.com/openshift/enhancements/blob/master/enhancements/monitoring/alerting-consistency.md#style-guide says `Alert names SHOULD be prefixed with a component, e.g.: AlertmanagerFailedReload`).